src/cljs/rems/user_settings.cljs

```diff
@@ -0,0 +1,22 @@
+(ns rems.user-settings
+  (:require [re-frame.core :as rf :refer [reg-event-fx reg-event-db]]
+            [rems.language :as language]
+            [rems.status-modal :as status-modal]
+            [rems.util :refer [fetch put!]]))
+
+(reg-event-fx
+ :loaded-user-settings
+ (fn [{:keys [db]} [_ user-settings]]
+   (language/update-language (:language user-settings))
```

Comment:
Need to check for a nil language. Currently this produces `<html lang="null">` when the user has no settings.

Reply:
Or then make it impossible to have a nil language. It there are no user settings, then some reasonable defaults should be returned.

Reply:
I ended up returning default settings from get-user-settings if they are unset. It seemed like the most robust solution. Thanks for the suggestion.